code/modules/virus2/effect.dm

```diff
@@ -1103,18 +1103,18 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
 			others_count += 1
 	if (others_count >= multiplier)
 		to_chat(mob, "<span class='notice'>A friendly sensation is satisfied with how many are near you - for now.</span>")
-		mob.adjustBrainLoss(-multiplier)
-		mob.reagents.add_reagent(PARACETAMOL, multiplier) // The hug chemical
+		mob.adjustCloneLoss(-multiplier / 5)
+		mob.reagents.add_reagent(PARACETAMOL, multiplier) // The hug chemical for memes
```

Comment:
Worthless. Not enough of a reward to justify following its need. Use something far more potent.